packages/widget/src/components/address-input/styles.ts

```diff
@@ -0,0 +1,36 @@
+import { css } from 'lit';
+
+export const styles = css`
+  .addressInputContainer {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 8px;
+  }
+
+  .inputAddressContainer {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 91px;
+    gap: 8px;
+  }
+
+  .inputAddress {
+    border-radius: 24px;
+    border: 1px solid var(--zinc-200, #e4e4e7);
```

Comment:
copy pasta from figma? 😂 

```suggestion
    border: 1px solid #e4e4e7;
```

also a suggestion: add `variables.ts` file with color variables and use them in files (follow Cindy's convention of naming them, e.g. `const zinc400 = '#e4e4e7'`)

Reply:
applied color definition to `:host` selector